index.js

```diff
@@ -72,6 +73,7 @@
 //. during development. For example:
 //.
 //. ```javascript
+//. //    def :: String -> StrMap (Array TypeClass) -> Array Type -> Function -> Function
```

Comment:
Shall we move this to the preceding code block, or include it in both places?

Reply:
I think both is a great idea.